sql/moz-fx-data-shared-prod/telemetry_derived/experiments_daily_active_clients_v1/metadata.yaml

```diff
@@ -2,7 +2,7 @@ friendly_name: Experiments Daily Active Clients
 description: |
   A daily number of active clients per experiment, partitioned by day.
 owners:
-- ssuh@mozilla.com
+- nobody@mozilla.com
```

Comment:
@scholtzan - Would you be an appropriate owner for `experiments_daily_active_clients` or have a suggestion for a new owner?

Reply:
yes, i can own this